torch/_inductor/fx_passes/node_runtime_estimation.py

```diff
@@ -68,19 +74,33 @@ def _benchmark_collective_with_cuda_events_impl(
     Core benchmarking logic using CUDA events and barriers.
     Returns runtime in ms or None on failure.
     """
-    import torch.distributed as c10d
+    from torch._dynamo.testing import rand_strided
+
+    # Convert FakeTensors to real tensors before benchmarking
+    def to_real(t: torch.Tensor) -> torch.Tensor:
+        shape = [get_hint(dim) for dim in t.shape]
+        stride = [get_hint(s) for s in t.stride()]
+
+        if any(s is None for s in itertools.chain(shape, stride)):
+            raise ValueError("Cannot convert tensor with symbolic dimensions")
```

Comment:
this shouldn't occur because can_benchmark_collective should have already checked unbacked.. maybe update error msg? / add comment